src/andromede/model/parsing.py

```diff
@@ -12,72 +12,79 @@
 import typing
 from typing import List, Optional
 
-from pydantic import BaseModel, Field
+from pydantic import BaseModel, Field, ValidationError
 from yaml import safe_load
 
 
 def parse_yaml_library(input: typing.TextIO) -> "InputLibrary":
     tree = safe_load(input)
-    return InputLibrary.model_validate(tree["library"])
+    try:
+        return InputLibrary.model_validate(tree["library"])
+    except ValidationError as e:
+        raise ValueError(f"An error occurred during parsing: {e}")
 
 
 # Design note: actual parsing and validation is delegated to pydantic models
 def _to_kebab(snake: str) -> str:
     return snake.replace("_", "-")
 
 
-class InputParameter(BaseModel):
-    name: str
-    time_dependent: bool = False
-    scenario_dependent: bool = False
+class ModifiedBaseModel(BaseModel):
+    description: Optional[str] = None
```

Comment:
We should not make this field common to all classes (all yaml parts), but instead add it in the few classes where it's really allowed:
- library
- model
- port type

For now that's all !